packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json

```diff
@@ -18,11 +18,12 @@
   "user_account_identity_documents_selection_file_format_invalid": "Le format {{values}} du ou des documents téléchargés n'est pas accepté. Veuillez respecter le format requis : jpg, jpeg, pdf et png",
   "user_account_identity_documents_submit": "Envoyer mes documents",
   "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents !",
```

Comment:
You missed an evolution here: the text should read:
"Merci, nous avons bien reçu tous vos documents !"

Reply:
Thank you for your reply. I fixed it.